src/App.tsx

```diff
@@ -1,8 +1,82 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
 import './App.css';
+import { getTodos } from './api/utils/getTodos';
+import { getUsers } from './api/utils/getUsers';
+import { TodoList } from './components/TodoList/TodoList';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+export const App = () => {
+  const [isLoading, setIsLoadind] = useState<boolean>(false);
+  const [preparedTodos, setPreparedTodos] = useState<PreparedTodo[]>([]);
+  const [typeOfSort, setTypeOfSort] = useState<string>('');
```

Comment:
примитивы не указываются
```suggestion
  const [isLoading, setIsLoadind] = useState(false);
  const [preparedTodos, setPreparedTodos] = useState<PreparedTodo[]>([]);
  const [typeOfSort, setTypeOfSort] = useState('');
```